### PR TITLE
Allow symfony security-csrf and asset 5.1

### PR DESCRIPTION
**Pull request by jordisala1991**
## Subject

I am targeting this branch, because this is BC.

## Changelog

```markdown
### Added
- Allow `symfony/security-csrf` and `symfony/asset` ^5.1
```

### CODE REVIEW

**Review by VincentLanglet — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -51,8 +51,8 @@
         "symfony/routing": "^4.4",
         "symfony/security-acl": "^3.1",
         "symfony/security-bundle": "^4.4",
-        "symfony/security-core": "^4.4",
-        "symfony/security-csrf": "^4.4",
+        "symfony/security-core": "^4.4 || ^5.1",
```
Comment (franmomu) [inline composer.json:54]: Looks like `symfony/security-core` is not `5.1`:

https://github.com/sonata-project/SonataAdminBundle/runs/1057543595#step:10:90

```bash
composer why-not symfony/security-core 5.1
symfony/security-bundle      v4.4.12  requires  symfony/security-core (^4.4)
```

**Comment by wbloszyk** [comment]
@jordisala1991 
Could you move this PR to master branch? We can't add support for Symfony5 in Sonata3 becouse [Symfony 5 - TODO list](https://github.com/sonata-project/SonataAdminBundle/issues/6365)

**Comment by jordisala1991** [comment]
This change does not break bc, so it can be done on 3.x (I will revert the security-core)

**Comment by franmomu** [comment]
> This change does not break bc, so it can be done on 3.x (I will revert the security-core)

Can you please add symfony/asset 😬 ? https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Asset/CHANGELOG.md apparently there is no problem.

**Comment by wbloszyk** [comment]
@jordisala1991 
I know this is not BC-break. I mean something else. [Some symfony bundle must be with symfony version](https://github.com/symfony/symfony/blob/master/composer.json#L40-L104). What is point to add support for `5.x` when install it will be imposible? That why I ask about move it to master.

**Comment by jordisala1991** [comment]
Because people usually do not have all possible symfony dependencies fixed on 4.4 or use flex like this:

```
    "extra": {
        "symfony": {
            "allow-contrib": true,
            "require": "^4.4 || ^5.1"
        }
    },
```

So if a package is compatible, I am happy with getting an upgrade. I don't care if I don't have all possible packages upgraded

**Review by franmomu — APPROVED** [review]

**Review by wbloszyk — APPROVED** [review]

**Comment by franmomu** [comment]
Thanks @jordisala1991!

**Pull request merged**